src/libsystemd-network/dhcp-option.c

```diff
@@ -35,6 +36,31 @@ static int option_append(uint8_t options[], size_t size, size_t *offset,
                 *offset += 1;
                 break;
 
+        case SD_DHCP_OPTION_USER_CLASS: {
+                size_t i = 0;
```

Comment:
the name `i` for something that accumulates values is still strange... `i` is usually used for iterators, but this isn't one

Reply:
renamed